main.go

```diff
@@ -21,6 +22,8 @@ const (
 )
 
 var configFile = flag.String("file", "", "Path to configuration file")
+var cpuprofile = flag.String("cpuprofile", "", "Write CPU profile to file")
+var memprofile = flag.String("memprofile", "", "write memory profile to this file")
```

Comment:
Wonder if it's worth mentioning these flags somewhere in the docs. I was actually looking for a way to change the config file path the other day 😄